lib/qa/authorities/getty/tgn.rb

```diff
@@ -0,0 +1,65 @@
+module Qa::Authorities
+  class Getty::TGN < Base
+    include WebServiceBase
+
+    attr_reader :subauthority
+    def initialize(subauthority)
+      @subauthority = subauthority
+    end
+
+    def search q
+      parse_authority_response(json(build_query_url(q)))
+    end
+
+    # get_json is not ideomatic, so we'll make an alias
+    def json(*args)
+      get_json(*args)
+    end
+
+    def build_query_url q
+      query = URI.escape(sparql(untaint(q)))
+      "http://vocab.getty.edu/sparql.json?query=#{query}&_implicit=false&implicit=true&_equivalent=false&_form=%2Fsparql"
+
+    end
+
+    def sparql(q)
+      search = untaint(q)
```

Comment:
Should more parsing be done here? I know it doesn't exist anywhere else but this current form is quite limited. For example, "Atlanta" doesn't give me the likely best match of "Atlanta, GA".

So maybe it might make sense to split the search term on either the first "," or "(" and trim that part of the string to match the gvp:prefLabelGVP. Then use the remainder to match the parent string via regex? This way one might be able to do "Atlanta, Georgia" or "Atlanta (Georgia" and get better results?

(May be a bad idea and wouldn't work for "Atlanta, GA")